report/src/problem_analysis/hci/test_results.tex

```diff
@@ -5,11 +5,11 @@ \section{Evaluation results}\label{sec:usability_evaluation_results}
 One of the problems described in Section \ref{sec:understanding_the_problem} explains that it is tedious to modify ongoing meetings. 
 During the test, it was found that the prototype was received positively in this regard. 
 Users found the interaction with and information on the prototype intuitive. 
-They understood the different situations depicted on the prototype and understood how to interact and complete the test tasks described in Appendix \ref{appendix:usability_tasks} without issues.
+They understood the different situations depicted on the prototype and understood how to interact with the prototype to complete the test tasks described in Appendix \ref{appendix:usability_tasks} without issues.
 During the debriefing, it was mentioned that the addition of being able to extend and modify meetings using a tablet would be an improvement to their current workflows.
 However, it was speculated whether other approaches to making these changes to meetings could be done. 
-The prototype focuses on the integration with existing systems already being used by the stakeholder and relies on user interaction for managing the meetings.
-Since both the prototype and their current system is fully dependent on users correctly managing the meetings, some aspect of the problematic situation cannot be solved by this product. 
+The prototype focuses on the integration with existing systems already in use by the stakeholder and relies on user interaction for managing the meetings.
+Since both the prototype and their current system is fully dependent on users correctly managing the meetings, some aspects of the problematic situation cannot be solved by this product. 
```

Comment:
```suggestion
Since both our prototype and their current system fully depend on users correctly managing the meetings, some problematic situations cannot be solved by this prototype. 
```